src/main/scala/viper/gobra/util/ChopperUtil.scala

```diff
@@ -6,17 +6,28 @@
 
 package viper.gobra.util
 
+import java.io.File
+import java.util.Properties
+
 import viper.silver.{ast => vpr}
 import viper.silver.ast.SourcePosition
 import viper.gobra.frontend.Config
-import viper.gobra.reporting.{ChoppedViperMessage, Source}
+import viper.gobra.reporting.ChoppedViperMessage
 import viper.gobra.backend.BackendVerifier.Task
 
 object ChopperUtil {
 
+  val GobraChopperFileLocation = "GobraChopper.conf"
```

Comment:
could you please add an example conf file somewhere? E.g. at Gobra's root directory?